provider/cluster/service.go

```diff
@@ -172,6 +172,7 @@ loop:
 			break loop
 
 		case ev := <-s.sub.Events():
+			s.log.Debug("event", "ev", ev)
```

Comment:
Double check: do you want to keep this in there? Might make `debug` very noisy, but it might be helpful.

Reply:
:shrug: it's not *too* bad when just running the provider.

I think we should add one at the source - in `./event/publish.go` tho so we don't have to do it all over the place.  I'll remove it from here.